ZAP_Chat_07_Automation_Framework_Part_1_english.srt

```diff
@@ -64,7 +64,7 @@ Great. And I'll move some of the toolbars out of my way so I can see what's goin
 
 17
 00:01:13,640 --> 00:01:18,180
-So what I wanted to show you is start with if we go to documentation.
+So what I wanted to show you is start with if we go to docentation.

```

Comment:
Removed um in other words.

Reply:
done

Reply:
so now it says documentation again

Reply:
There were several words no just this one: https://github.com/zaproxy/zap-media-resources/pull/4/commits/4c63d3b0639d12f238ad3db92e0709689d3171e6

Reply:
ok no idea why this happened now -.- I'll look over it again later